src/ontology/mondo-ingest-odk.yaml

```diff
@@ -33,6 +33,8 @@ components:
       source: https://data.bioontology.org/ontologies/ICD10CM/submissions/23/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb
     - filename: icd10who.owl
       source: https://github.com/monarch-initiative/icd10who/releases/latest/download/icd10who.ttl
+    - filename: icd11foundation.owl
```

Comment:
## `build-mondo-ingest`
- [ ] @joeflack4 

Running it now.

Reply:
This is a very important issue, and will have to be reported, documented and discussed with the WHO team.. I would suggest:

1. In your ICD11 preprocessing code, add a ROBOT subcommand `remove --axioms equivalents`
2. Create a report for all these inferred equivalents (just the relevant lines from the reasoner error output), and 
3. Use `robot explain` (see docs) to generate an explanation for two or three of the equivalents and post them into that issue as well.
4. Document that design decision carefully in the icd11foundation.md docs

When we meet with WHO, we will raise the issue with them.

Reply:
@matentzn This sounds like a good plan, thanks!

Regarding (1), can you confirm I have the right idea?
It seems like doing this in `mondo-ingest` wouldn't be right, because this problem occurs during the steps in `Makefile`, which is not a file we are supposed to edit. So instead, I'd do this change in the ICD11 repo (https://github.com/monarch-initiative/icd11/pull/2), after the command that removes unicode characters: `robot remove --axioms equivalents -i PATH -o release $@`.

Reply:
Yeah I think thats ok. Go ahead as you are saying and I will review the PR!

Reply:
(make sure the issue is clearly documented in the mondo ingest docs for icd11)

Reply:
@twhetzel @matentzn Build is passing!
I would imagine it's safe to merge this, but feel free to look over it some more.
There are some remaining tasks in the review comments (see checkboxes). I intend to finish those tomorrow.

Happy because this is the first time I've fully ran a successful build.

<details><summary>Log snippet</summary>
<p>

```
Finished running all tests successfully.
...
Release files are now in ../.. - now you should commit, push and make a release         on your git hosting site such as GitHub or GitLab
rm reports/icd11foundation_exclusion_reasons.robot.template.tsv slurp/icd11foundation.tsv imports/ro_terms_combined.txt
make[1]: Leaving directory '/work/src/ontology'
echo "Mondo Ingest has been fully completed"
Mondo Ingest has been fully completed
make: Leaving directory '/work/src/ontology'
```

</p>
</details>

Reply:
If we merge this please migrate my instructions above to an issue - this will be important for our communication with WHO.

Reply:
I've created the issue, and will check this box when those tasks Nico outlined above are completed.
- [ ] @joeflack4 complete tasks in https://github.com/monarch-initiative/icd11/issues/4